backend/CircleForms/Extensions/PostIdFormatExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace CircleForms.Extensions;
+
+public static class PostIdFormatExtensions
+{
+    public static string ToPostId(this string id)
```

Comment:
I'm not a big fan of using extensions for this (or at least extending strings), but it can work for now.